easybuild/tools/options.py

```diff
@@ -615,6 +615,8 @@ def config_options(self):
                                     None, 'store', None),
             'installpath-software': ("Install path for software (if None, combine --installpath and --subdir-software)",
                                      None, 'store', None),
+            'installpath-data': ("Install path for data (if None, combine --installpath and --subdir-data)",
+                                 None, 'store', None),
```

Comment:
Please move this to before `installpath-modules` to maintain the alphabetical order.

Reply:
fea2ae1